content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/ups_credentials.rst

```diff
@@ -1,68 +1,210 @@
-complete the registration process.
+Shipping account
+----------------
 
-If you already have a UPS account, you can add it to your online profile as follows:
+To add an Odoo shipment account, select :guilabel:`Add New Account` from the :guilabel:`Add a
+Payment Method` drop-down menu, and click :guilabel:`Add`.
 
-1. Log in to the `UPS.com website <http://www.ups.com/>`_) using your UPS.com User ID and Password.
+.. image:: ups_credentials/new-account.png
+   :align: center
+   :alt: Display "Add an account" option from the drop-down menu.
 
-2. Click the **My UPS** tab.
+Next, complete the forms to configure company details and address for setting up the shipping
+account.
 
-3. Click the **Account Summary** link.
+.. image:: ups_credentials/shipping-account.png
+   :align: center
+   :alt: Display UPS form to fill out company shipping information.
 
-4. Click the **Add an Existing UPS Account** link in the **UPS Account Details**
-section of the page.
+.. _inventory/shipping_receiving/ups-account-number:
 
-5. In the **Add New Account** screen, enter the **UPS Account Number**,
-**Account Name**, and **Postal Code** fields. The country defaults to
-United States.q
+Get account number
+------------------
 
-6. Click the **Next** button to continue.
+With the shipping account set up, the UPS :guilabel:`Account Number` becomes available. To access
+it, navigate to :menuselection:`Profile --> Accounts and Payment` and refer to the shipping
+account's :guilabel:`Number` field.
 
-Get an Access Key
-=================
+.. image:: ups_credentials/account-number.png
+   :align: center
+   :alt: Show Account "Number" field for the shipping account.
+
+Payment card
+------------
+
+Navigate back to the :guilabel:`Accounts and Payments` page and select the :guilabel:`Add Payment
+Card` option from the :guilabel:`Add a Payment Method` drop-down menu. Then, complete the form to
+add the credit card information.
+
+.. image:: ups_credentials/payment-card.png
+   :align: center
+   :alt: Show "Add Payment Card" option from the drop-down.
+
+UPS developer account setup
+===========================
+
+Next, log into the `UPS developer account <http://developer.ups.com/>`_ to generate the developer
+key. To begin, click the profile icon in the top-right corner, and choose the :guilabel:`Apps`
+option from the drop-down menu.
+
+.. image:: ups_credentials/apps.png
+   :align: center
+   :alt: Display the "Apps" drop-down option after clicking the profile picture icon.
+
+Add app
+-------
+
+Then, click the :guilabel:`Add Apps` button to begin filling out the form. In the :guilabel:`I need
+API credentials because` field, select :guilabel:`I want to integrate UPS technology into my
+business`.
+
+In :guilabel:`Choose an account to associate with these credentials`, select :guilabel:`Add existing
+account` and select the :ref:`account number <inventory/shipping_receiving/ups-account-number>`
+linked to the UPS account created in the previous step.
```

Comment:
specificity
also, clarification, do we *select* the account (is a pre-filled option) or do we have to add it ourselves (i.e. via text input)?
```suggestion
Under the next label, :guilabel:`Choose an account to associate with these credentials. \*`, select
:guilabel:`Add existing account` from the drop-down menu in the corresponding field, and then select
the :ref:`account number <inventory/shipping_receiving/ups-account-number>` linked to the UPS
account created in the previous step.
```

Reply:
Since I wasn't able to test this part, I don't know whether we select the account number or type it :sweat_smile: